opencog/atoms/base/Handle.h

```diff
@@ -300,6 +300,8 @@ std::string oc_to_string(const HandleUCounter& huc, const std::string& indent);
 std::string oc_to_string(const HandleUCounter& huc);
 std::string oc_to_string(Type type, const std::string& indent);
 std::string oc_to_string(Type type);
+std::string oc_to_string(const TypeSet& types, const std::string& indent);
```

Comment:
Is it important to have the indent-version in the public API? Can these just be moved to the .cc file, so they are not in the .h?

Reply:
I'm afraid we do need them in the .h because other oc_to_string overloading not living within that file (like the onces in the URE or the pattern miner) want to have access to the indent versions. And we also need the non-indent version in the .h for gdb...